src/zmalloc.c

```diff
@@ -383,8 +381,6 @@ static void *zrealloc_pmem(void *ptr, size_t size) {
     newptr = realloc_pmem(ptr,size);
     if (!newptr) zmalloc_oom_handler(size);
 
-    update_zmalloc_pmem_stat_free(oldsize);
```

Comment:
Why you remove this USE_MEMKIND_MEMTIER consider this in ifdef block AFAIK this block is shared with USE_MEMKIND

Reply:
I think it's not shared with a USE_MEMKIND block as there is a IF ELSE statement choosing between USE_MEMKIND_MEMTIER block and USE_MEMKIND block. Am I missing sth?

Reply:
It is ok